src/main/java/com/sleekbyte/tailor/utils/Pair.java

```diff
@@ -0,0 +1,41 @@
+package com.sleekbyte.tailor.utils;
+
+/**
+ * Couples together a pair of values, which may be of different types (L and R).
+ * The individual values can be accessed through its public members first and second.
```

Comment:
Javadoc should be updated since it has conflicting information. It should probably state: "…accessed via its public getter functions `getFirst()` and `getSecond()`".